core/src/main/java/com/sequenceiq/cloudbreak/conf/ScheduleConfig.java

```diff
@@ -1,31 +1,47 @@
 package com.sequenceiq.cloudbreak.conf;
 
+import java.util.concurrent.ExecutorService;
+
+import javax.inject.Inject;
+
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.scheduling.TaskScheduler;
 import org.springframework.scheduling.annotation.EnableScheduling;
 import org.springframework.scheduling.annotation.SchedulingConfigurer;
 import org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler;
 import org.springframework.scheduling.config.ScheduledTaskRegistrar;
 
 import com.sequenceiq.cloudbreak.concurrent.MDCCleanerThreadPoolTaskScheduler;
 
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.binder.jvm.ExecutorServiceMetrics;
+
 @Configuration
 @EnableScheduling
 public class ScheduleConfig implements SchedulingConfigurer {
 
+    public static final String SCHEDULER_TASK_EXECUTOR = "SCHEDULER_TASK_EXECUTOR";
```

Comment:
What is the reason for mixed uppercase and lowercase to this value?

Reply:
I will use this as the bean name of the task scheduler just like elsewhere.